content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -79,7 +79,37 @@ $ dvc exp run --all-pipelines
 > Note that the order in which pipelines are executed is not guaranteed; Only
 > the internal order of stage execution is.
 
-### Interactive reproduction
+### Running pipelines recursively
+
+DVC supports pipelines defined in more than one `dvc.yaml` file. These can
+reside in subfolders inside the workspace, and you may want to run all of them
+at once. Example project:
```

Comment:
> Continues https://github.com/iterative/dvc.org/pull/2846#pullrequestreview-763545319

To summarize on this, `exp run --recursive` has to do with how your project structure is organized so in case people have several dvc.yaml files they may need it. I don't think we need to mention pipelines to explain this, even when they user may be using them indeed — they would know what they are and how/where they codified them already.

That said if you want to mention both the cases with multiple dvc.yaml files and the one with multiple pipelines as part of the explanation here, that's OK too. We just shouldn't focus on the concept of pipelines I believe.

Reply:
Another option or an additional thing we may want to do is to create a separate "Organizing Experiments" page (instead of https://dvc.org/doc/user-guide/experiment-management#organization-patterns) and mention `exp run --recursive` as a tool in there. This could be a separate issue or check box somewhere. Up to you @iesahin

Reply:
This section is about "running experiments" and I believe it involves experiments that use pipelines in directory hierarchies. We can mention `--recursive` in other places as a way to organize the experiments, but _here_ we are discussing _how to run experiments_ if their parts are found in recursive directories.

Reply:
Pipelines are an inherent part of experiments, even if we have "experiments without pipelines" at some point, this guide will have to cover "experiments with pipelines." Specifying pipelines, organizing them or adding/deleting stages to them are irrelevant topics to this section but "running pipelines" should be covered here IMO. If we'd remove all "pipeline related stuff" from this section, about 90% of the material should be deleted. We don't have any means to run the experiments other than running the pipelines. 

This individual section tells how to _run_ the experiments _if_ their parts are organized in directory hierarchies, not _organizing_ experiments into directory hierarchies.

Reply:
> here we are discussing how to run experiments if their parts are found in recursive directories

Right, it makes sense to mention `exp --recursive` here. I just don't think we need a full section that describes the project's organization. It can be a paragraph (maybe even a single sentence) in the existing **Running all pipelines** section.

> Pipelines are an inherent part of experiments
at some point, this guide will have to cover "experiments with pipelines."

Disagree. You don't need a pipeline for experiments, you just need a dvc.yaml file. It's more of a technicality and in the future this implementation will be made transparent with `exp init` anyway. But we've discussed this a lot already...

> If we'd remove all "pipeline related stuff" from this section, about 90% of the material

Most would stay, just explained from a different perspective, as mentioned in https://github.com/iterative/dvc.org/pull/2846#discussion_r718142001.

Reply:
> Disagree. You don't need a pipeline for experiments, you just need a dvc.yaml file.

In the current implementation, we need a pipeline at least with a single stage to run an experiment. Even `dvc exp init` will create a _default pipeline_ (`dvc.yaml` with some defaults) and run it.